### PR TITLE
Map moves and zooms

### PR DESCRIPTION
**Pull request by CaseyHillers**
Not the smoothest zoom yet, but I want to get this into the main branch so I can see if it works with the latest changes to the map.

### CODE REVIEW

File: app/js/functions.js [hunk 1]
```diff
@@ -193,12 +193,7 @@ function parseHash(fieldsById) {
   }
 
 //Inital map setup
-var map = d3.select("#map"),
```
Comment (BrightSun95) [inline app/js/functions.js:196]: Are the rest of these variables just not needed?

Comment (CaseyHillers) [reply app/js/functions.js:196]: When I removed the others, it broke the map :(

Removing just that one kept the map running, saved on an operation. just stuff I found out while figuring out the d3 and map.

**Review by BrightSun95 — COMMENTED** [review]
Looks fine to me but Ima leave it to somebody who knows more about d3 and the map to merge it lol.

**Review by larocque492 — APPROVED** [review]
Hey Casey, this looks good to me - I cross referenced this with a few examples on bl.ocks.org and all the d3 functions look like they do the right stuff. It'll be cool to see it in action. :)

**Pull request merged**